compiler/haskell-ide-core/src/Development/IDE/Functions/Compile.hs

```diff
@@ -414,7 +415,9 @@ getModSummaryFromBuffer fp (contents, fileDate) dflags parsed = do
           { ml_hs_file  = Just fp
           , ml_hi_file  = replaceExtension fp "hi"
           , ml_obj_file = replaceExtension fp "o"
+#ifndef USE_GHC
```

Comment:
Is this a `ghc` vs `ghc-lib` or an `8.6` vs `8.8` thing? What do we think is the default backend, `ghc` or `ghc-lib`? I think we should make the default backend work without any pragma.

Reply:
It's more an 8.6 vs 8.8 thing. We do need to figure out which is the default, and probably want to use the Cabal package-ghc version macro at some point. For the moment, I tried to change as little in our code, but assuming one day this goes into a separate repo, then it probably wants to be the other way around.